### PR TITLE
add project structure

### PR DESCRIPTION
**Pull request by napalmpapalam**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+FROM node:14-alpine3.11 as builder
```
Comment (napalmpapalam) [inline Dockerfile:1]: node 14 old release and not maintained anymore use `node:18-alpine`

File: README.md [hunk 2]
```diff
@@ -1,49 +1,72 @@
...
+## Changelog
+
+For the changelog, see [CHANGELOG.md](./CHANGELOG.md).
+  
 ## License
 
-This project is licensed under the MIT License - see the [LICENSE.md](./LICENSE) file for details
+This project is under the MIT License — see the [LICENSE](./LICENSE) file for details.
+
+[Rarimo Proof of Humanity]: https://docs.rarimo.com/use-cases/proof-of-humanity
```
Comment (napalmpapalam) [inline README.md:66]: -

File: README.md [hunk 3]
```diff
@@ -1,49 +1,72 @@
...
+
+[Download Here]: https://nodejs.org/en/download/
+
+[Instructions to Download]: https://www.typescriptlang.org/download
+
+["build a custom action"]: https://dev.collab.land/docs/upstream-integrations/build-a-custom-action
```
Comment (napalmpapalam) [inline README.md:72]: remove

File: package.json [hunk 4]
```diff
@@ -0,0 +1,78 @@
+{
+  "name": "rarime-dashboard",
+  "private": true,
+  "version": "1.0.0",
```
Comment (napalmpapalam) [inline package.json:4]: 0.1.0

File: package.json [hunk 5]
```diff
@@ -0,0 +1,78 @@
+{
+  "name": "rarime-dashboard",
```
Comment (napalmpapalam) [inline package.json:2]: add description and 

```
"license": "MIT",
  "author": "Zero Block Global Foundation",
  "keywords": [
    "rarime"
  ],
  "repository": {
    "type": "git",
    "url": "https://github.com/rarimo/rarime-dashboard.git"
  },
```

File: package.json [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+    "build": "dotenv -e .env.production yarn vite-build",
+    "analyze": "dotenv -e .env.analyze yarn vite-build",
+    "lint": "yarn lint:styles && yarn lint:scripts",
+    "lint:styles": "stylelint \"src/**/*.{css,sass,scss}\" --max-warnings=0",
+    "lint:scripts": "eslint \"{src,config}/**/*.{js,ts,jsx,tsx}\" --cache --fix --max-warnings=0",
+    "preview": "vite preview",
+    "rsc": "node scripts/release-sanity-check.mjs"
+  },
+  "dependencies": {
+    "@distributedlab/tools": "^0.2.0-rc.10",
```
Comment (napalmpapalam) [inline package.json:22]: use latest version

File: scripts/release-sanity-check.mjs [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+    'i',
+  )
+
+  if (releaseTagIsNotTopRe.test(CHANGELOG_MD_CONTENT)) {
+    issuesFound.push(`The ${VERSION} is not the top tag in CHANGELOG.md`)
+  }
+}
+
+function validateChangelogAnchorsLegend () {
+  const baseRepoUrl = 'https://gitlab.com/distributed_lab/frontend/react-template'
```
Comment (napalmpapalam) [inline scripts/release-sanity-check.mjs:70]: __

File: src/enums/decimals.enum.ts [hunk 8]
```diff
@@ -0,0 +1,3 @@
+export enum WEB3_DECIMALS {
+  WEI = 18,
```
Comment (napalmpapalam) [inline src/enums/decimals.enum.ts:2]: not needed BN already has this

File: src/enums/icon-names.enum.ts [hunk 9]
```diff
@@ -0,0 +1,232 @@
+export enum ICON_NAMES {
+  academicCap = 'academic-cap',
+  adjustments = 'adjustments',
```
Comment (napalmpapalam) [inline src/enums/icon-names.enum.ts:3]: I think u need to delete all this icons

Comment (WhiteNik16) [reply src/enums/icon-names.enum.ts:3]: fixed

File: src/enums/decimals.enum.ts (hunk 8 above)
Comment (WhiteNik16) [reply src/enums/decimals.enum.ts:2]: fixed

File: scripts/release-sanity-check.mjs (hunk 7 above)
Comment (WhiteNik16) [reply scripts/release-sanity-check.mjs:70]: fixed

File: package.json (hunk 6 above)
Comment (WhiteNik16) [reply package.json:22]: fixed

File: package.json (hunk 5 above)
Comment (WhiteNik16) [reply package.json:2]: fixed

File: package.json (hunk 4 above)
Comment (WhiteNik16) [reply package.json:4]: fixed

File: Dockerfile (hunk 1 above)
Comment (WhiteNik16) [reply Dockerfile:1]: fixed

**Pull request merged**